.github/workflows/pr-builder.yaml

```
@@ -5,11 +5,4 @@ jobs:
   run:
     runs-on: ubuntu-latest
     steps:
-      - uses: upsidr/merge-gatekeeper@v1
-        with:
-          self: "pr-builder / run"
-          token: ${{ secrets.GITHUB_TOKEN }}
-          interval: 60
-          timeout: 86400 # 24h
-          ref: ${{ github.ref }}
-          ignored: "triage,Branch Checker,Label Checker"
+      - run: exit 0
```

Maybe add a comment to explain how this is supposed to work.
```suggestion
      # This workflow should depend on all other workflows in a repository using "needs:"
      - run: exit 0
```